### DISCUSSION TITLE
iOS blank screen after app reactivate

### DISCUSSION THREAD

**Discussion by chrslgr** [airsdk/Adobe-Runtime-Support#2858] [General]
Hi.
I'm currently having the problem, that on iOS after my app goes back to the foreground (after screen sleep or other app was in foreground), my app only shows a blank screen. When i tap where a button on the view was and i hit that the view shows its contents again.
Has anyone had a similar issue or knows where I should look to find the reason for that problem?
Thanks a lot,
Chris

**Comment by raresn** [comment]
Some of my usewrs are actually experiencing the same on Android. Can't put my finger on it - we used to use the distriqt ANE with "black screen helper", but i feel like this is something new. Hope we can all join together and find some scenarios to replicate.
What os did you try on Chris? Does it happen each time?

**Reply by marchbold** [reply]
@raresn If you can isolate it to a device or android version or manage to replicate it I can help look into it too

**Comment by chrslgr** [comment]
The problem occurs on iOS (only as far as i know), built with Air 50.2.2.6. My device runs iOS 17.0.2, but the problem occurred on customers' devices with earlier versions (both Air and iOS) too.
It does not happen all the time, but quite often.
The app is quite simple by now, does not run fullscreen, uses some ANEs:

<extensionID>com.distriqt.Core</extensionID>
<extensionID>com.distriqt.Application</extensionID>
<extensionID>com.distriqt.BluetoothLE</extensionID>
<extensionID>com.distriqt.Scanner</extensionID>
<extensionID>com.distriqt.Share</extensionID>
<extensionID>com.myflashlab.air.extensions.dependency.overrideAir</extensionID>
<extensionID>com.myflashlab.air.extensions.permissionCheck</extensionID>
<extensionID>androidx.core</extensionID>
<extensionID>com.distriqt.playservices.AppSet</extensionID>
<extensionID>com.distriqt.playservices.Base</extensionID>

i could invite you to my testflight-app? would just need your apple connect email...

**Comment by Jason0247** [comment]
I have also encountered this issue before, both on Android and iOS.
At first, I thought it was a program issue, but when I saw that you also had the same problem, I guess it shouldn't be a program issue, right?
